tenable/sc/__init__.py

```diff
@@ -430,6 +432,14 @@ def recast_risks(self):
         '''
         return RecastRiskAPI(self)
 
+    @property
+    def reports_definition(self):
```

Comment:
Should be using a more correct name that conforms to the rest of the package

```suggestion
    def report_definitions(self):
```